findspam.py

```diff
@@ -915,6 +915,18 @@ class FindSpam:
         {'regex': r"car\Win", 'all': False, 'sites': ['superuser.com', 'puzzling.stackexchange.com'],
          'reason': 'bad keyword in {}', 'title': False, 'body': False, 'username': True, 'stripcodeblocks': False,
          'body_summary': False, 'max_rep': 1, 'max_score': 0},
+        # Judaism etc troll, 2018-04-18 ("potentially bad" makes this watch)
```

Comment:
> makes this **a** watch

?

Reply:
Actually I thought of it as a verb, but nvm.